### ISSUE TITLE
Version 0.9.3 reports selector depth greater than <limit> in-error

### ISSUE THREAD

**Issue by uglow** [rossPatton/stylint#89]
```css
//
// Fonts for Bootstrap
//

// Import the fonts - using styles produced from fontsquirrel.com
@font-face
  font-family: 'Corpid Heavy'
  src: url($icon-font-path + 'corpidc1s-heavy-webfont.eot')
  src: url($icon-font-path + 'corpidc1s-heavy-webfont.eot?#iefix') format('embedded-opentype'),
       url($icon-font-path + 'corpidc1s-heavy-webfont.woff2') format('woff2'),
       url($icon-font-path + 'corpidc1s-heavy-webfont.woff') format('woff'),
       url($icon-font-path + 'corpidc1s-heavy-webfont.ttf') format('truetype'),
       url($icon-font-path + 'corpidc1s-heavy-webfont.svg#CorpidC1sHeavyRegular') format('svg')
  font-style: normal
  font-weight: 500

...
```

Console log:
```console
Warning:  selector depth greater than 

Warning:  3:
File: src/modules/core/styles/component/fonts.styl
Line: 10: url($icon-font-path + 'corpidc1s-heavy-webfont.woff2') format('woff2'), 

Warning:  selector depth greater than 

Warning:  3:
File: src/modules/core/styles/component/fonts.styl
Line: 11: url($icon-font-path + 'corpidc1s-heavy-webfont.woff') format('woff'), 

Warning:  selector depth greater than 

Warning:  3:
File: src/modules/core/styles/component/fonts.styl
Line: 12: url($icon-font-path + 'corpidc1s-heavy-webfont.ttf') format('truetype'), 

Warning:  selector depth greater than 

Warning:  3:
File: src/modules/core/styles/component/fonts.styl
Line: 13: url($icon-font-path + 'corpidc1s-heavy-webfont.svg#CorpidC1sHeavyRegular') format('svg') 
```

Config:
```json
{
  "alphabetical": false,
  "borderNone": true,
  "brackets": false,
  "colons": false,
  "colors": false,
  "commaSpace": true,
  "commentSpace": false,
  "cssLiteral": false,
  "depthLimit": 3,
  "duplicates": false,
  "efficient": true,
  "enforceBlockStyle": false,
  "enforceVarStyle": true,
  "extendPref": "@extend",
  "globalDupe": true,
  "indentSpaces": 2,
  "leadingZero": false,
  "maxWarnings": 50,
  "maxWarningsKill": false,
  "mixed": true,
  "namingConvention": "BEM",
  "parenSpace": false,
  "placeholders": false,
  "semicolons": false,
  "universal": true,
  "valid": true,
  "whitespace": true,
  "zeroUnits": true,
  "zIndexDuplicates": false,
  "zIndexNormalize": false
}
```

Stylint Version 0.9.4

Also, the line break in the warning message makes it harder to understand the error.

Sorry for the trouble, Ross. I'll revert to 0.9.2 again.

**Comment by rossPatton** [comment]
hmm, ill look into it.

could try just using the `// @stylint off` comment before the `@font-face` for now

```
// @stylint off
@font-face
  font-family: 'Corpid Heavy'
  src: url($icon-font-path + 'corpidc1s-heavy-webfont.eot')
  src: url($icon-font-path + 'corpidc1s-heavy-webfont.eot?#iefix') format('embedded-opentype'),
       url($icon-font-path + 'corpidc1s-heavy-webfont.woff2') format('woff2'),
       url($icon-font-path + 'corpidc1s-heavy-webfont.woff') format('woff'),
       url($icon-font-path + 'corpidc1s-heavy-webfont.ttf') format('truetype'),
       url($icon-font-path + 'corpidc1s-heavy-webfont.svg#CorpidC1sHeavyRegular') format('svg')
  font-style: normal
  font-weight: 500
// @stylint on
```

**Comment by uglow** [comment]
Hi Ross, 

This particular error was fixed in v0.9.6, IIRC. But now I'm getting an invalid property error on the same @font-face block. I'll try to resolve, but otherwise I'll raise a separate issue.

Cheers,
Brett
